src/web-component.html

```diff
@@ -43,6 +43,7 @@
       // sidebar
       webComp.setAttribute("with_projectbar", "true");
       webComp.setAttribute("with_sidebar", "true");
+      webComp.setAttribute("react_app_api_endpoint", "https://test-editor-api.raspberrypi.org");
```

Comment:
🔧  I'd be tempted removing this from the test page and letting it get set in CI still
```suggestion
```